canonical-web-and-design/maas-ui
# refactor: subnets/space build delete space form

## Done

- build delete space form
- refactor index url to accept search param 

## Screenshots
![Kapture 2022-01-28 at 15 46 17](https://user-images.githubusercontent.com/7452681/151567447-7a24a8aa-9d45-4b61-aed8-db5a808b1c9a.gif)



## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Try to delete a space by going to the new url, e.g. `MAAS/r/space/1`.
- Verify everything worked correctly.

## Fixes

Fixes:  https://github.com/canonical-web-and-design/app-tribe/issues/646

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in master.
